core/src/main/java/com/linecorp/armeria/client/metric/ClientMeterIdPrefixFunction.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *  Copyright 2017 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+package com.linecorp.armeria.client.metric;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.logging.RequestLog;
+import com.linecorp.armeria.common.logging.RequestOnlyLog;
+import com.linecorp.armeria.common.metric.MeterIdPrefix;
+import com.linecorp.armeria.common.metric.MeterIdPrefixFunction;
+import com.linecorp.armeria.common.metric.MeterIdPrefixFunctionCustomizer;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+
+/**
+ * Creates a {@link MeterIdPrefix} for a client from a {@link RequestLog}.
+ *
+ * @see MetricCollectingClient
+ */
+public interface ClientMeterIdPrefixFunction extends MeterIdPrefixFunction {
```

Comment:
Let's consider it when we reach 2.0 because 
- Users can still use the `MeterIdPrefixFunction` as is. `ClientMeterIdPrefixFunction` doesn't provide anything special if users don't want to add raddr.
- I didn't want users to fix their existing code.

Reply:
In that case, I slightly prefer that `MeterIdPrefixFunction#ofClientDefault`, `MeterIdPrefixFunction#ofClientDefaultBuilder` is introduced instead since the additional interface doesn't provide much benefit.

Having said this, let me leave this decision to you since I think this is a trivial point.

Reply:
Alternatively, `MetricCollectingClientBuilder` could have an `instanceof` check on the accepted interface so that at least users are aware of misconfiguration at build time

Reply:
>  I slightly prefer that MeterIdPrefixFunction#ofClientDefault,

That was my actually my first thought, but @trustin suggested this way.
I'm okay either way. Let's talk about it at the standup, today. 😉